scripts/config/mt-gateway.ini

```diff
@@ -0,0 +1,63 @@
+
+#http service address
+addr = :5059
+
+#graphite api address
+graphite-url = http://localhost:8080
+
+#mt-whisper-importer-writer address
+#importer-url =
+
+#metrictank address
+metrictank-url = http://localhost:6060
+
+
+#default org ID to send to downstream services if none is provided (ignored if value = -1)
+default-org-id = -1
+
+
+###
+### The properties below are for configuring ingestion to kafka via the `/metrics` endpoint
+###
+
+#enable metric publishing
+metrics-publish = false
+
+#path to carbon storage-schemas.conf
+schemas-file = /etc/metrictank/storage-schemas.conf
+
+#kafka tcp address(es) for metrics, in csv host[:port] format
```

Comment:
wouldn't `csv` mean that it's ok to have spaces after the `,`? Do we allow spaces after the `,`?

Reply:
How do you feel about dc6254a242c77e3545ba92b3d931ce5b7f4e79af ?

Worst case we should fail fast and they can figure it out from the error message

Reply:
I'd just say it the same was as in the metrictank config, for consistency and clarity: https://github.com/grafana/metrictank/blob/master/metrictank-sample.ini#L271

Reply:
0c55a44ee243c3794318a43f597ef6abac45fd3d